src/main/java/carpet/script/api/WorldAccess.java

```diff
@@ -673,7 +683,104 @@ else if (v instanceof EntityValue)
             });
             return (cc, tt) -> result[0];
         });
+        // =======================================
+        expression.addContextFunction("structure_block_save", -1, (c, t, lv) -> {
+            CarpetContext cc = (CarpetContext) c;
+            ServerWorld lv2 = cc.s.getWorld();
```

Comment:
why do you name your variables like lv2, lv3, so on? It kinda makes it hard to know what is what when reading the code